### DISCUSSION TITLE
Where you get this ui, i see it in stripe aslo

### DISCUSSION THREAD

**Discussion by salahhusa9** [wire-elements/wire-spy#33] [Q&A] [answered]
https://github.com/wire-elements/wire-spy/blob/main/.github/bar.png

**Comment by salahhusa9** [comment]
@PhiloNL ??

**Reply by PhiloNL** [reply, marked as answer]
The UI colors and layout are inspired by Stripe's workbench but made from scratch using Tailwind.
